README.md

```diff
@@ -43,11 +42,24 @@ See the command line options with:
 
 ### SSL/TLS
 
-`feed-ingress` expects your ELBs to terminate SSL traffic. We believe this is the safest and best performing
+## SSL termination on ELB
+SSL termination could be done on ELBs, probably the safest and best performing
```

Comment:
"SSL termination could be done on ELBS, and we believe that this is the safest and best performing"